web_app/src/components/trips/trip-list.tsx

```diff
@@ -0,0 +1,43 @@
+import React, {useEffect} from 'react'
+import TripSummary from './trip-summary'
+import {Link} from 'react-router-dom'
+import {useStoreActions, useStoreState} from "../../hooks"
+
+export default function TripList() {
+  const {items, error, loading} = useStoreState(state => state.trips)
```

Comment:
Vær forsiktig med å hente ut hele `state.trips` for så å deconstructe. Hvis `state.trips` er komplisert (som den kommer til å bli) vil dette føre til at denne komponenten re-rendres hver gang noe som helst endrer seg i `state.trips`. Jeg vil kalle det best practice å alltid hente hver verdi ut fra storen hver for seg. `easy-peasy` memoiserer verdien sånn at ingen re-rendering vil skje hvis verdien ikke endrer seg.

```suggestion
  const items = useStoreState(state => state.trips.items)
  const error = useStoreState(state => state.trips.error)
  const loading = useStoreState(state => state.trips.loading)
```